src/snyk/snykCode/codeSettings.ts

```diff
@@ -43,6 +44,7 @@ export class CodeSettings implements ICodeSettings {
     }
     await this.contextService.setContext(SNYK_CONTEXT.CODE_ENABLED, codeEnabled);
     await this.contextService.setContext(SNYK_CONTEXT.CODE_LOCAL_ENGINE_ENABLED, localCodeEngineEnabled);
+    this.config.setFeatureFlag(FEATURE_FLAGS.consistentIgnores, codeEnabled);
```

Comment:
I'm not sure if this is the right place to put it, ideally we would have a more global way of updating feature flag status even if this is for Snyk Code

Reply:
You're right, this is not the correct implementation.
Working on it 💪